viewer/views/layout.jade

```diff
@@ -171,6 +178,13 @@ html
         - if (molochClusters)
           each key in Object.keys(molochClusters)
             command.sessionSendSessionButton.sendSessionAction(cluster=key,label="Send Session to #{molochClusters[key].name}")
+        - if (user.fragments && user.fragments.session)
+          - var keys = Object.keys(user.fragments.session).sort()
+          - if (keys.length > 0)
+            span 
+              b Reports
+            each key in keys
+              command#sessionReport(label="#{user.fragments.session[key].name}", data-filename="#{user.fragments.session[key].identifier}")
```

Comment:
when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
command(class="sessionReport", label="#{user.fragments.session[key].name}", data-filename="#{user.fragments.session[key].identifier}")